framework/materials/multi_group_xs/openmc_import.cc

```diff
@@ -96,7 +96,7 @@ MultiGroupXS::Initialize(const std::string& file_name,
     for (int g = 0; g < num_groups_; ++g)
       for (int gp = g_min[g]; gp <= g_max[g]; ++gp)
         for (int n = 0; n < scattering_order_ + 1; ++n, ++fidx)
-          transfer_matrices_.at(n).Insert(g, gp - 1, flat_scatter_matrix[fidx]);
+          transfer_matrices_.at(n).Insert(gp - 1, g, flat_scatter_matrix[fidx]);
```

Comment:
This is why a serious spectral verification is needed when implementing new features. Both the scattering and the production matrices were transposed, but this does not change the eigenvalue. I can add a brief comment if you want.

Reply:
So then my statement is correct? OpenMC stores scattering with rows designating the scattered to group and columns the scattered from groups? It may be more clear to make the first for loop `gp` and the second `g` if that is the case, especially since `g_min`/`g_max` are not `gp_min`/`gp_max`.

Reply:
yes! I was already working on this. Same for the production!